src/screens/OtherProfile/index.js

```diff
@@ -603,7 +528,8 @@ const OtherProfile = () => {
   };
 
   const handleDataFeed = () => {
-    const isFeedsShown = !blockStatus.blocked && !blockStatus.blocker;
+    if (isProfileFetching) return [];
+    const isFeedsShown = !(isBlocking || isBlocked);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for showing feeds has been updated. Make sure that the new conditions (`isBlocking || isBlocked`) are equivalent to the old condition (`blockStatus.blocked && blockStatus.blocker`) in this context.



<!-- This is an auto-generated comment by OSS CodeRabbit -->